internal/e2e/sds_test.go

```diff
@@ -169,12 +169,12 @@ func TestSDSShouldNotIncrementVersionNumberForUnrelatedSecret(t *testing.T) {
 	// TODO(dfc) 1166: currently Contour will rebuild all the xDS tables
```

Comment:
please delete the todo, this PR fixes it.

Reply:
👍